cmd/node/config/config.toml

```diff
@@ -233,6 +233,16 @@
    MinTimeToWaitBetweenBroadcastsInSec = 20
    MaxTimeToWaitBetweenBroadcastsInSec = 25
    DurationInSecToConsiderUnresponsive = 60
+   [Heartbeat.HeartbeatStorage]
+       [Heartbeat.HeartbeatStorage.Cache]
+           Size = 100000
+           Type = "LRU"
+       [Heartbeat.HeartbeatStorage.DB]
+           FilePath = "HeartbeatStorage"
+           Type = "LvlDBSerial"
+           BatchDelaySeconds = 15
+           MaxBatchSize = 45000
```

Comment:
Set to 100.

Reply:
Warning: since the MaxBatchSize is now 100, the app will wait either to have 100 hbmi's to write or wait for 15 seconds. This actually reduce the accuracy of the stored data in case of a node's failure as all data prepared to be written are actually delayed by a time frame. Is this intended?

Reply:
Ok! Changed to 1. I thought there are more than 100 nodes that will send heartbeats at more than 20 seconds and the writes are done at at most 15 seconds so just 15 seconds of data is lost, if it is lost. Changed to 1, because the heartbeats are not that intensive.